tests/libcxx/host/host.cpp

```diff
@@ -280,7 +280,11 @@ int main(int argc, const char* argv[])
     }
 
     // Disable stdout buffering on host
+#ifdef _WIN32
+    setvbuf(stdout, NULL, _IONBF, 0);
```

Comment:
Why not just use `setvbuf` in both cases here?

Reply:
Ah I missed this one. Will update it shortly.